docker-compose.test.yml

```
@@ -0,0 +1,29 @@
+    restart: unless-stopped
+    volumes:
+      - .:/app
+      - /app/.next
+    ports:
+      - '3001:3001'
+
+  playwright:
+    build:
+      context: .
+      dockerfile: Playwright.Dockerfile
+    command: [ "pnpm", "run", "${TEST_TYPE:-test:e2e}" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)